protocol/x/clob/keeper/process_single_match.go

```diff
@@ -307,6 +310,7 @@ func (k Keeper) persistMatchedOrders(
 ) (
 	takerUpdateResult satypes.UpdateResult,
 	makerUpdateResult satypes.UpdateResult,
+	affiliateRevSharesQuoteQuantums *big.Int,
```

Comment:
Seems more generic to return `types.RevSharesForFill` below

Reply:
This one specifically is for specifically sending an event which only using affiliateRevShares. Making it generic would require us to iterate through the list or updating revshares object in some way